pom.xml

```
@@ -610,6 +605,16 @@
         <artifactId>jline</artifactId>
         <version>3.20.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <version>5.8.2</version>
+      </dependency>
+      <dependency>
+        <groupId>org.junit.vintage</groupId>
+        <artifactId>junit-vintage-engine</artifactId>
+        <version>5.8.2</version>
+      </dependency>
```

It might be possible to manage all the junit dependencies with a single bom:

```suggestion
      <dependency>
        <groupId>org.junit</groupId>
        <artifactId>junit-bom</artifactId>
        <version>5.8.2</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
```

---

addressed in [466348b](https://github.com/apache/accumulo/pull/2427/commits/466348b9a2e23f776be47c54b236545d2b62805c)